src/foam/nanos/NanoService.js

```diff
@@ -14,6 +14,18 @@ foam.INTERFACE({
       javaThrows: [
         'java.lang.Exception'
       ]
+    },
+    {
+      name: 'reload',
+      args: [
+        {
+          name: 'x',
+          type: 'Context'
+        }
+      ],
+      javaCode: `
+        //add implementation
+      `
```

Comment:
NanoService is an interface, did you mean to have default implementation for the reload() method?

Reply:
Yes, as I thought our interfaces supported this now.

Reply:
yes, default method implementation on the interface(?). What is wrong with this

Reply:
It failed to build but on the second look seems like you forget to pass x to the reload call.

Reply:
yes, my bad forgot the context. But we do support the default interface implementation